app/Http/Controllers/WebhookController.php

```diff
@@ -122,11 +122,17 @@ public function changeStatus(Request $request)
         }
 
         $result = $this->webhookRepository->update($request->id, ['status' => $status]);
-        
+
         if ($result) {
             return 'This webhook was updated successfully';
         }
 
         return 'Updated failed. Something went wrong';
     }
+
+    public function adminIndex () {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 133</br>Opening brace should be on a new line